package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": ">=18.17.1"
+    "node": ">=22.13.1"
```

Comment:
Note that if you're adding this line, then the package will give errors for anyone using a lower version of Node.js. Is this version a minimum requirement now? (i.e. to we want to require anyone using this project to be on node 22.13.1 or later?)

Reply:
As per discussion with @minestarks last Friday, yes.
> If you mean v22 (the default one that's offered on the download page) then yes, I believe that's what we should use.

Reply:
Tagging @idavis on this one. I believe our ADO pipeline uses the Node.js that comes with the image. Our last ADO run points to this image <https://github.com/actions/runner-images/blob/ubuntu22/20250120.2/images/ubuntu/Ubuntu2204-Readme.md> which lists Node 18.20.5 (and I don't believe we install a later version).

I don't think our project should require a later version than our pipelines run with.